packages/qdrant-loader-core/src/qdrant_loader_core/logging_filters.py

```
@@ -20,6 +20,45 @@ def filter(self, record: logging.LogRecord) -> bool:
+                    redacted = self._SENSITIVE_QUERY_PARAM.sub(
+                        r"\1***REDACTED***", path
+                    )
+                    if redacted != path:
+                        record.args = (
+                            record.args[0],
+                            record.args[1],
+                            redacted,
+                            *record.args[3:],
+                        )
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)